array-filter/main.js

```diff
@@ -0,0 +1,34 @@
+const numbers = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const names = [
+  'Ada',
+  'Hedy',
+  'Jean',
+  'Grace',
+  'Evelyn',
+  'Joan',
+  'Elizabeth',
+  'Janese',
+  'Donna'
+];
+
+const evenNumbers = numbers.filter(number => number % 2 === 0);
+console.log(evenNumbers);
+
+const overFive = numbers.filter(number => number > 5);
+console.log(overFive);
+
+const startWithE = names.filter(name => name.startsWith('E'));
+console.log(startWithE);
+
+function haveDFunction(name) {
+  if (name.includes('d')) {
+    return true;
+  } else if (name.startsWith('D')) {
+    return true;
```

Comment:
`haveDFunction` should return the same value no matter where in the name a capital `'D'` is.